package.json

```diff
@@ -1,22 +1,34 @@
 {
   "name": "@derekstride/tree-sitter-sql",
-  "version": "0.3.0",
+  "version": "0.3.1",
   "description": "Tree-sitter Grammar for SQL",
   "main": "bindings/node",
+  "types": "bindings/node",
   "scripts": {
     "test": "tree-sitter test",
-    "install": "tree-sitter generate && node-gyp rebuild",
-    "release": "commit-and-tag-version"
+    "install": "tree-sitter generate && node-gyp-build",
+    "release": "commit-and-tag-version",
+    "prebuildify": "prebuildify --napi --strip"
   },
   "author": "derek stride",
   "license": "MIT",
   "dependencies": {
-    "nan": "^2.18.0"
+    "node-addon-api": "^7.1.0",
+    "node-gyp-build": "^4.8.0"
   },
   "devDependencies": {
     "commit-and-tag-version": "^12.0.0",
     "node-gyp": "^10.0.1",
-    "tree-sitter-cli": "^0.20.8"
+    "prebuildify": "^6.0.0",
+    "tree-sitter-cli": "^0.22.6"
```

Comment:
Good that we upgrade. However it will break the tests for now, since the queries in the tests are interpreted in a different order (I don't remember if first or last query wins)  regardless, we have to make changes in the tests.

Reply:
Yes, tested and it inversed the type and function.call. I'm fixing it.

Reply:
fixed. The highlight evaluation is switched, so I changed it and now tests are passing.